lib/tools/adb-commands.js

```diff
@@ -231,7 +231,6 @@ methods.sendTelnetCommand = async function (command) {
         dataStream += data;
         if (readyRegex.test(data)) {
           res = dataStream.replace(readyRegex, "").trim();
-          res = _.last(res.trim().split('\n'));
```

Comment:
Yes with gulp test i was not able to run my code on emulator.But still tested using the adb commands directly through terminal.That works on almost all Android APIs.